app/src/main/java/com/pixlee/pixleeandroidsdk/ui/gallery/KtxGalleryListFragment.kt

```diff
@@ -214,6 +227,26 @@ class KtxGalleryListFragment : BaseFragment(), LifecycleObserver {
         }
     }
 
+    override fun uploadFile(filePath: String) {
+        viewModel.uploadPhoto(
+                localMediaPath = filePath,
+                title = "uploaded from SDK-" + System.currentTimeMillis() + " using a file",
```

Comment:
hardcodes?

Reply:
This is the demo app area and is to demonstrate how the SDK can be used. I'll replace this with a sample data.